src/test/java/com/wikia/webdriver/common/logging/Log.java

```diff
@@ -116,8 +122,10 @@ public static void logOnLowLevel(String command, String description, boolean suc
     log(command, description, success, true);
   }
 
-  private static void log(String command, String description, boolean isSuccess,
-                          boolean ifLowLevel) {
+  private static void log(
+      String command, String description, boolean isSuccess,
+      boolean ifLowLevel
+  ) {
     LOGS_RESULTS.add(isSuccess);
     String escapedDescription = escapeHtml(description);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "escapedDescription" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1481)